package.json

```diff
@@ -40,39 +40,35 @@
   },
   "devDependencies": {
     "@polymer/iron-component-page": "^4.0.0",
-    "@typescript-eslint/eslint-plugin": "^4.29.2",
-    "@typescript-eslint/parser": "^4.29.2",
+    "@typescript-eslint/eslint-plugin": "^4.31.2",
+    "@typescript-eslint/parser": "^4.31.2",
     "@vaadin/testing-helpers": "^0.3.0",
-    "@web/dev-server": "^0.1.22",
-    "@web/rollup-plugin-html": "^1.9.1",
+    "@web/dev-server": "^0.1.23",
+    "@web/rollup-plugin-html": "^1.10.1",
     "@web/test-runner": "^0.13.17",
     "@web/test-runner-commands": "^0.5.12",
     "@web/test-runner-playwright": "^0.8.8",
     "@web/test-runner-saucelabs": "^0.6.2",
     "@web/test-runner-visual-regression": "^0.6.3",
-    "axios": "^0.21.1",
+    "axios": "^0.21.4",
     "dotenv": "^10.0.0",
     "eslint": "^7.32.0",
     "eslint-config-prettier": "^8.3.0",
-    "eslint-plugin-prettier": "^3.4.1",
+    "eslint-plugin-prettier": "^4.0.0",
     "glob": "^7.1.7",
     "husky": "^4.3.8",
     "lerna": "^4.0.0",
     "lint-staged": "^11.1.2",
     "npm-run-all": "^4.1.5",
-    "octokit": "^1.0.4",
-    "playwright": "^1.15.0-next-alpha-sep-13-2021",
```

Comment:
Hmm, what package provides `playwright` now?

Reply:
It is a transitive dependency of `@web/test-runner-playwright`.

Reply:
Ah, I got it. Thanks!